win32/vcxproj/DSGame-server.vcxproj

```diff
@@ -23,6 +23,7 @@
     <ProjectGuid>{89A19910-84DB-454D-9C7F-0A47A293867B}</ProjectGuid>
     <RootNamespace>DSGameserver</RootNamespace>
     <Keyword>Win32Proj</Keyword>
+    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>

```

Comment:
this is uneeded and was probably auto generated by having vs itself modify file while being used on win10 machine so it applied the default for your setup.

Reply:
when omitted the build just uses whatever you are "on" anyway.